spikeinterface/comparison/tests/test_templatecomparison.py

```diff
@@ -24,7 +24,6 @@ def setup_module():
 
 
 def test_compare_multiple_templates():
-    test_dir = Path("temp_comp_test")
```

Comment:
I think someone left this in a previous debugg session and was generating files in the wrong place.